packages/@aws-cdk/assert/package.json

```
@@ -53,7 +53,7 @@
   "peerDependencies": {
     "@aws-cdk/core": "0.0.0",
     "constructs": "^3.3.69",
-    "jest": "^26.6.3"
+    "jest": ">=26.6.3"
```

After discussing with @RomainMuller, I take my previous comment back. The correct value here should be

```suggestion
    "jest": ">=26.6.3 <28.0.0"
```

---

Applied